src/member/migrations/0026_change_to_emergency_contacts.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-01-26 16:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def migrate_emergency_contacts(apps, schema_editor):
+    # set NotePriority initial
+    Client = apps.get_model('member', 'Client')
+    EmergencyContact = apps.get_model('member', 'EmergencyContact')
+    for client in Client.objects.all():
+        if client.emergency_contact:
+            EmergencyContact.objects.create(
+                client=client,
+                member=client.emergency_contact,
+                relationship=client.emergency_contact_relationship,
+            )
+
+
+def reverse_emergency_contacts(apps, schema_editor):
+    # set NotePriority initial
+    Client = apps.get_model('member', 'Client')
+    for client in Client.objects.all():
+        # Use as client emergency contact the first item of
+        # relationship
+        emergency_contact = client.emergencycontact_set.first()
+        # reversing info
+        client.emergency_contact = emergency_contact.member
+        client.emergency_contact_relationship = emergency_contact.relationship
+        # removing info
+        client.emergencycontact_set.delete()
```

Comment:
I've the same problem with `clear` method, finnaly I changed `client.emergencycontact_set.delete()` by `EmergencyContact.objects.filter(client=client).delete()`